Monika After Story/game/dev/dev_unittest.rpy

```diff
@@ -2734,3 +2735,43 @@ label dev_unit_test_strict_can_pickle:
     call dev_unit_tests_finish_test(scp_tester)
 
     return
+
+label dev_unit_test_mas_set_pronouns:
+    m "Running tests..."
+
+    $ pronouns_tester = store.mas_dev_unit_tests.MASUnitTester()
+
+    python hide:
+        pronouns_tester.prepareTest("Validate map size and keys")
+        map_size = None
+        for sub_map in store.MAS_PRONOUN_GENDER_MAP.values():
+            sub_map = dict(sub_map)
+
+            if map_size is None:
+                map_size = len(sub_map)
+            else:
+                pronouns_tester.assertTrue(map_size == len(sub_map))
+
+            pronouns_tester.assertTrue("M" in sub_map)
+            pronouns_tester.assertTrue("G" in sub_map)
+            pronouns_tester.assertTrue("X" in sub_map)
```

Comment:
these are checking if the sub map is the same length as other maps, and the main 3 gender types have an entry in the sub map, correct? to make it easier for debugging, we should have a prepare test line for the size check, and then a prepare test line before the gender types checks. Can use the pronoun (key) in the prepare test text as well. That way when Monika says a test failed, it won't always be "Validate map size and keys" when only one part of the map has an issue.